source/_data/glossary.yml

```diff
@@ -1,13 +1,15 @@
 - topic: Action
-  description: "[Actions](/docs/automation/action/) are events that fires once all triggers and conditions have been met."
+  description: "An [Action](/docs/automation/action/) is an event that can be fired as a response to a trigger,  once all conditions have been met."
+- topic: Add-on
+  description: "Hass.io add-ons, often referred to as just: Add-ons, provide additional services that can be installed in a Hass.io ecosystem. For example, an MQTT broker, database service or a file server. Not to be confused with integrations. Integrations connect Home Assistant with devices and service, add-ons can provide those services."
```

Comment:
Should we use the word services or applications? I think that if we use "standalone applications" it might be more clear.

Add-ons provide additional, standalone, applications that can run besides Home Assistant on a hass.io system. These applications are then integrated into Home Assistant using integrations. Examples of add-ons are an MQTT broker, database service or a file server.

Reply:
Adjusted to: 

> Hass.io add-ons provide additional, standalone, applications that can run beside Home Assistant on a Hass.io installation. Most of these, add-on provided, applications can be integrated into Home Assistant using integrations. Examples of add-ons are: an MQTT broker, database service or a file server.